source/dynamical_systems/include/dynamical_systems/Linear.hpp

```diff
@@ -16,6 +16,8 @@
 #include "state_representation/space/cartesian/CartesianState.hpp"
 #include "state_representation/space/cartesian/CartesianTwist.hpp"
 
+using namespace state_representation;
```

Comment:
I am not certain we want that in header files. In source file this is not a problem because it can't break much but in header files this basically mean that any file that includes the header will also use this namespace. I prefer that we use a verbose solution than to do that.